azkaban-web-server/src/web/js/azkaban/view/job-list.js

```diff
@@ -43,6 +44,20 @@ azkaban.JobListView = Backbone.View.extend({
     this.list = $(this.el).find("#joblist");
     this.contextMenu = settings.contextMenuCallback;
     this.listNodes = {};
+
+    this.presetToggleBtn("autoPanZoom");
+    this.presetToggleBtn("autoExpand");
+  },
+
+  presetToggleBtn: function(name) {
+    var target = "#" + name + "Btn";
+    if (localStorage.getItem("LiAz-" + name) !== 'false') {
```

Comment:
If defined in the azkaban namespace (`$.namespace('azkaban')`) can the prefix for locally stored variables be shared across files? Or, is there a way to avoid using a hardcoded prefix across different js files?

Reply:
Given the program structure, it's not simple to achieve. The new code complexity would negate the benefit of a common variable. The new variable name would have to be used in multiple files anyway. Not worth it.